src/error.rs

```diff
@@ -139,6 +140,9 @@ pub enum Error {
         "Network not found, check config.toml, specify network with -n flag"
     )]
     NetworkNotFound,
+    /// Trying to perform an operation on a poisioned mutex
+    #[error("A Mutex was poisioned. Please wait a little to perform the next operation")]
+    PoisionError(()),
```

Comment:
What is the reason for passing an empty tuple here? Given you don't use the data coming from `std::sync::PoisonError`, you can just leave it as a unit variant.